packages/legend-query/src/stores/QueryBuilderLambdaProcessor.ts

```diff
@@ -385,11 +385,20 @@ export class QueryBuilderLambdaProcessor
   ): void {
     const functionName = valueSpecification.functionName;
     if (matchFunctionName(functionName, SUPPORTED_FUNCTIONS.GET_ALL)) {
+      const _class = valueSpecification.genericType?.value.rawType;
+      let acceptedNoOfParameters = 1;
+      if (_class instanceof Class && _class.stereotypes.length !== 0) {
+        acceptedNoOfParameters = valueSpecification.parametersValues.length;
+        assertTrue(
+          acceptedNoOfParameters === 2,
+          'Milestoning class should have a parameter',
+        );
+        this.queryBuilderState.getAllFunctionState = valueSpecification;
+      }
       assertTrue(
-        valueSpecification.parametersValues.length === 1,
+        valueSpecification.parametersValues.length === acceptedNoOfParameters,
         `Can't process getAll() expression: getAll() expects no argument`,
       );
-      const _class = valueSpecification.genericType?.value.rawType;
       assertType(
```

Comment:
Lets move this assert of Type further up, before we check if its a milestone class.